test/fixtures/users.yml

```diff
@@ -0,0 +1,11 @@
+# Read about fixtures at https://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
```

Comment:
how did you cerated fixtures file? automatically or you added by yourself?

Reply:
The fixtures were added automatically